### ISSUE TITLE
Provide hash values of nano binaries

### ISSUE THREAD

**Issue by slasheks** [LiskHQ/lisk-nano#451]
### Expected behaviour
Include a hash for the binaries in the download section of lisk nano to compare what was downloaded to what is expected/provided by LiskHQ. 

### Actual behaviour
Have to trust it is what is expected. No way to compare. 

### Steps to reproduce
n/a

Would be nice to have them pgp signed as well. Example: http://keepass.info/integrity.html

**Comment by Isabello** [comment]
Windows and osx binaries are signed with a certificate issues to the Lisk
Stiftung. I would say thats better than hashes. Can add hashes for linux
though

On Jul 1, 2017 16:49, "slasheks" <notifications@github.com> wrote:

Expected behaviour

Include a hash for the binaries in the download section of lisk nano to
compare what was downloaded to what is expected/provided by LiskHQ.
Actual behaviour

Have to trust it is what is expected. No way to compare.
Steps to reproduce

n/a

Would be nice to have them pgp signed as well. Example:
http://keepass.info/integrity.html

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/LiskHQ/lisk-nano/issues/451>, or mute the thread

**Comment by slasheks** [comment]
Yes, a certificate, given that is not compromised is better than a hash to provide verification of integrity but not of corruption in transfer.
